evap/staff/templates/staff_evaluation_operation.html

```diff
@@ -53,11 +53,59 @@
                 {% endif %}
             </div>
         {% endif %}
-        <input type="hidden" name="target_state" value="{{ target_state }}" />
+        <input type="hidden" name="target_state" value="{{ target_state }}"/>
+
+        {% if show_delete_answers_checkbox %}
+            <div class="card mb-3">
+                <div class="card-body">
+                    <div class="form-check">
+                        <input class="form-check-input" type="checkbox" name="delete-previous-answers" id="delete-previous-answers" checked>
+                        <label class="form-check-label" for="delete-previous-answers">
+                            {% trans "Delete previous answers" %}
+                        </label>
+                    </div>
+                </div>
+            </div>
+        {% endif %}
 
         <div class="card card-submit-area card-submit-area-2 text-center mb-3">
             <div class="card-body">
-                <button type="submit" class="btn btn-primary">{% translate 'Confirm' %}</button>
+            {% if show_delete_answers_checkbox %}
+                <confirmation-modal type="submit" form="evaluation-operation-form" id="confirmation-modal">
```

Comment:
Since this element is already nested in the `evaluation-operation-form`, the `form=` attribute is not needed :)